client/modules/wj-state/src/media.ts

```diff
@@ -1,9 +1,14 @@
-import { writable } from "svelte/store"
+import { Readable, writable } from "svelte/store"
+
+export type BreakpointName = "narrow" | "small" | "normal" | "wide"
+export type BreakpointString = `${">" | "<" | ">=" | "<=" | ""}${BreakpointName}`
```

Comment:
I don't think I understand what "breakpoints" mean here.

Reply:
Breakpoint is a common term for the pixel sizes (e.g. screen which is 400 pixels across, which isn't real pixels but weird CSS relative pixels), where the CSS changes when that size is crossed.

Reply:
Ohh right, I was thinking about parsing and wondering if this was some editor thing I didn't understand. I was thinking about syntax and forgot `@media` queries existed.